ultraplot/axes/plot.py

```diff
@@ -2296,6 +2296,9 @@ def _parse_cmap(
         cmap_kw = cmap_kw or {}
         norm_kw = norm_kw or {}
         # If norm is given we use it to set vmin and vmax
+        if (vmin is not None or vmax is not None) and norm is not None:
+            print(vmin, vmax, norm)
```

Comment:
This print statement needs to be removed.

Reply:
done was too quick to ping you